runtime/lua/editorconfig.lua

```diff
@@ -190,6 +190,29 @@ function properties.insert_final_newline(bufnr, val)
   end
 end
 
+--- A code of the format ss or ss-TT, where ss is an ISO 639 language code and TT is an ISO 3166 territory identifier.
+--- Sets the 'spell' and 'spelllang' options.
```

Comment:
I am not sure how I feel about this also setting `'spell'`. Someone may want to apply `spelling_language` globally (using a `[*]` glob) so that _when they do_ enable spell checking, it will use the specified language. But that does not imply they necessarily want spell checking always enabled.

I think we should remove `'spell'` for now and set only `'spelllang'`. This is reversible if we need to re-enable it later.

Reply:
The spec also doesn't say anything about enabling spell checking.
> Sets the natural language that should be used...